examples/tutorials/translations/português/Parte 06 - Aprendizado Federado com MNIST usando uma CNN.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>I just realized that I forgot a paragraph repeated and with the original English version.</p><p> </p><blockquote>You should now be able to do Federated Learning by yourself! If you enjoyed this and would like to join the movement toward privacy preserving, decentralized ownership of AI and the AI supply chain (data), you can do so in the following ways! </blockquote><blockquote>Agora você deve poder fazer Aprendizado Federado sozinho! Se você gostou disso e gostaria de se juntar ao movimento em direção à preservação da privacidade, propriedade descentralizada da IA e da cadeia de suprimentos da AI (dados), você pode fazê-lo das seguintes maneiras! </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Resolved!